tests/mock_classes/mock_question_filter.py

```diff
@@ -0,0 +1,18 @@
+"""Mocked QuestionFilter."""
+
+
+class MockQuestionFilter:
```

Comment:
why do we need a mock for this?  Seems pretty straightforward to create the actual object